cypress/e2e/court-cases-index/unlocking.cy.ts

```diff
@@ -94,30 +103,50 @@ describe("Case unlocked badge", () => {
     loginAndGoToUrl()
 
     // Exception lock
-    cy.get(`tbody tr:nth-child(1) .locked-by-tag`).get("button").contains("Bichard Test User 01").should("exist")
-    cy.get(`tbody tr:nth-child(1) img[alt="Lock icon"]`).should("exist")
+    cy.get("tbody")
```

Comment:
this quite complicated check that the lock tag exists gets repeated a few times, we could extract it to a helper function like is done for `unlockCase` to make changes to it easier